indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -72,11 +73,15 @@ def _process_args(self):
 
     @use_rh_args(EditableFileTypeArgs, partial=True)
     def _process_PATCH(self, data):
+        if 'publishable' in data and not data['publishable']:
+            check_if_can_unset_or_delete(self.event, self.file_type)
```

Comment:
Can you try whether you can put this check in the schema instead of having it here?

Reply:
I put it on the schema as discussed in MM but that doesn't cover the case of deletion, right? Should I leave the function in `operations.py` and call it both in the schema and `management.py`?